build.sbt

```diff
@@ -82,16 +82,18 @@ def jdk11GcJavaOptions: Seq[String] = {
   )
 }
 
+// Upgrading to a later version requires also upgrading sbt-scoverage, Scalatest, and
+// some modules like util-mock and util-test.
 val _scalaVersion = "2.13.6"
-val _crossScalaVersions = Seq("2.12.12", "2.13.6")
+val _crossScalaVersions = Seq("2.12.15", "2.13.6")
 
 val defaultScalaSettings = Seq(
   scalaVersion := _scalaVersion,
   crossScalaVersions := _crossScalaVersions
 )
 val defaultScala3EnabledSettings = Seq(
   scalaVersion := _scalaVersion,
-  crossScalaVersions := _crossScalaVersions ++ Seq("3.0.2-RC1")
+  crossScalaVersions := _crossScalaVersions ++ Seq("3.1.3")
```

Comment:
There is a merge conflict because a PR was merged that changed this to 3.0.2 - building with 3.0.2 is better imho because classes will work with Scala 3.0.x and 3.1.x. If you compile with 3.1.3, then the classes will not work with Scala 3.0.x runtime.

Reply:
Are you sure about that? I thought binary compatibility was finally solved by the Tasty mechanism. 

The libraries published to Maven also don't specify anything beyond "_3" as the version suffix.

If there is really still a problem I don't mind building with 3.0.x for the moment, though.

Reply:
Ok, I see. That's unfortunate.

I will rework/close my PRs as soon as I find the time.